### DISCUSSION TITLE
Similar questions as #1093 (pseudobulk DE analysis)

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1272] [Feature Requests]
Hi @rcorces,

I am also interested in using pseudo-bulk replicates for differential analysis. I know getMarkerFeatures can be used but it is not possible to control for covariates with wilcox test, for example. Moreover, with getGroupSE() its not possible to get pseudo-bulk replicates per cluster that were generated during peak calling. 

Is it possible to add a function that simply exports/returns the pseudo-bulk replicates created during addGroupCoverages? This will be extremely helpful to maintain consistency of insertion bed file that user may have to regenerate if starting from getFragmentsFromArrow().

Thank you,
Anjali

**Comment by rcorces** [comment]
Better differential analysis is on our radar. You can create labels in cellColData that define your desired pseudobulk reps and then use getGroupSE() to pull this out.

The feature you are requesting already exists - see the `returnGroups` parameter:
https://www.archrproject.com/reference/addGroupCoverages.html
